x-pack/plugin/security/qa/profile/build.gradle

```diff
@@ -9,7 +9,7 @@ dependencies {
   javaRestTestImplementation project(':x-pack:plugin:security')
 }
 
-boolean literalUsername = (new Random(Long.parseUnsignedLong(BuildParams.testSeed.tokenize(':').get(0), 16))).nextBoolean()
+boolean literalUsername = BuildParams.random.nextBoolean()
```

Comment:
@mark-vieira I am not sure whether adding a static `BuildParams#getRandom` method is the right way to go. So I only change this one place to show case where and how it is needed. If you approves this approach, I'll search and replace similar usages through the code base. Thanks!

Reply:
It's probably not worth introducing another utility for this. This seems fine to me.